packages/terra-cli/index.js

```diff
@@ -0,0 +1,119 @@
+const yargs = require('yargs/yargs');
+const path = require('path');
+const fs = require('fs-extra');
+
+const Logger = require('./lib/utils/Logger');
+
+const logger = new Logger({ prefix: '[terra-cli:terra]' });
+
+const DEPENDENCY_ALLOW_LIST = ['@cerner/terra-functional-testing', '@cerner/terra-open-source-scripts'];
+
+/**
+ * Does the initial set up of the CLI. This sets up things like help, version, etc.
+ */
+const setupCLI = () => {
+  const cli = yargs();
+
+  return cli.usage('Usage: $0 <command> [options]')
+    .demandCommand(1, 'A command is required. Pass --help to see all available commands and options.')
+    .recommendCommands()
+    .strict()
+    .fail((msg, err) => {
+      // certain yargs validations throw strings
+      const actual = err || new Error(msg);
```

Comment:
nitpick, can we rename this to something more specific, like `actualErr`

Reply:
Updated: https://github.com/cerner/terra-toolkit/pull/460/commits/5314365d74c3ef7d44104a23b5d3143723176f6d